hydrus/app.py

```diff
@@ -27,6 +27,20 @@ def failed_authentication() -> Response:
                                     headers=[{'WWW-Authenticate': 'Basic realm="Login Required"'}])
     return response
 
+def check_authentication():
```

Comment:
You have to use python3 type annotation here.
Specify argument types and return type.
As your function is not taking any arguments you have to just specify the return type.